spc/spcserver.py

```diff
@@ -475,12 +426,34 @@ def convert_date(date):
     def _download(self, server, img_url):
         ''' Downloads image to desired output directory
 
-        :param img_url: string parsed from json_data during retrieval
+        :param img_url: list containing strings parsed from json_data during
+            retrieval
         :return:
         '''
-        srcpath = self.imgurl.format(server, img_url)
-        destpath = self.output_dir.format(os.path.basename(img_url))
-        urllib.urlretrieve(srcpath, destpath)
+        assert isinstance(img_url, list), 'Store Image URLs in list'
+        for ii, url in enumerate(img_url):
+            srcpath = self.imgurl.format(server, url)
+            destpath = self.output_dir.format(os.path.basename(url))
+            urllib.request.urlretrieve(srcpath, destpath)
```

Comment:
This is a really slow method for downloading images as it is done serially; a multithreaded approach should work a lot faster.